intelmq/bots/collectors/bitsight/collector.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+
+import re
+import sys
+import pycurl
+from intelmq.lib.bot import Bot
+from intelmq.lib.message import Report
+
+
+class BitsightCollectorBot(Bot):
+
+      def init(self):
+          self.logger.info("Connecting to BitSightTech stream server") 
+          http_proxy = self.parameters.http_proxy 
+          https_proxy = self.parameters.http_ssl_proxy 
+          self.conn  = pycurl.Curl()
+          if http_proxy:
+             self.conn.setopt(pycurl.PROXY, str(http_proxy))
+          if https_proxy:
+             self.conn.setopt(pycurl.PROXY, str(https_proxy))
```

Comment:
"Are both http_proxy and https_proxy necessary?"  No, I use just http. Just added for future needs.
"If one of them is not given, does the bot still work?" Yes.

But, let me make some more testes.